File: SellerDroidSolution/SellerDroidWeb/Models/DTO/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SellerDroidWeb.Models.DTO
{
    public class Product : ProductSummary
    {
        public String longDescription { get; set; }

        public Supplier supplier { get; set; }
    }
}

File: SellerDroidSolution/SellerDroid/app/src/main/java/lb/edu/aust/sellerdroid/dto/Product.java
package lb.edu.aust.sellerdroid.dto;

/**
 * AUST Student on 18/12/2015.
 */
public class Product extends ProductSummary {
    public String longDescription;
    public Supplier supplier;

    public Product(String productName, String productDescription) {
        super(productName, productDescription);
    }
}
